backend/src/main/scala/cromwell/backend/BackendWorkflowInitializationActor.scala

```diff
@@ -27,6 +32,60 @@ object BackendWorkflowInitializationActor {
   */
 trait BackendWorkflowInitializationActor extends BackendWorkflowLifecycleActor with ActorLogging {
 
+  /**
+    * Answers the question "does this expression evaluate to a type which matches the predicate".
```

Comment:
I tried it initially. It needed a whole other set of `NoFunctions` and would have involved evaluating a value and then throwing it away anyway. Also nobody else uses it so in the end I thought I might as well just do it like this